src/disemvowel.js

```diff
@@ -0,0 +1,17 @@
+import readlineSync from 'readline-sync'
+
+const disemvowel = ( string ) => {
```

Comment:
i like how you wrote out the whole function for disemvowel. Another way this could be done is with regex ;)